controllers/users.js

```diff
@@ -7,13 +7,13 @@ const getPixelatedImage = async (image) => {
   const pixelatedImage = await originalImage
     .pixelate(10)
     .getBase64Async(Jimp.MIME_JPEG);
-  res.render("users/register.ejs", { image });
   return pixelatedImage;
 };
 
 module.exports.renderRegister = async (req, res, next) => {
+  const extraStyles = '<link rel="stylesheet" href="/stylesheets/cards.css" />';
```

Comment:
Ok this is nice, interesting idea. I would recommend to declare the constant outside the module export again because I see the same string in a route below.

Reply:
Is what i initially tried to do but had issues with, although it could have been because i was passing the variable to the .ejs incorrectly, will check it out!

Reply:
Ok well let me know how it goes, however it should work just fine like this: `{ extraStyles: $variable }`